.github/workflows/ci.yml

```diff
@@ -41,11 +41,13 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-20.04"]
-        elixir: ["1.15", "1.14", "1.13"]
+        elixir: ["1.18", "1.16", "1.15", "1.14", "1.13"]
```

Comment:
I think you can drop `1.13`, it was marked end of life https://endoflife.date/elixir

Reply:
I'll update, thanks!